ci-operator/step-registry/acm/tests/alc/acm-tests-alc-commands.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# The variables defined in this step come from files in the `SHARED_DIR` and credentials from Vault.
+SECRETS_DIR="/tmp/secrets"
+
+# Get the creds from ACMQE CI vault and run the automation on pre-exisiting HUB
+#SKIP_OCP_DEPLOY=$(cat $SECRETS_DIR/ci/skip-ocp-deploy)
```

Comment:
I would  just delete this line and lets remove `skip-ocp-deploy` from vault.

When a user needs to make use of an external cluster they will just edit line 11 in their PR.